.gitignore

```diff
@@ -4,3 +4,5 @@
 auditjs.json
 # ci config for local ci build
 /.circleci/local-config.yml
+
+.nyc_output
```

Comment:
TRIVIAL: newline

Reply:
Lol, in: https://github.com/sonatype-nexus-community/auditjs/pull/146/commits/6d3d7fbb3b2c2066a870c6ec2d92b0f5b09770d7